src/averageAges.js

```diff
@@ -37,7 +43,19 @@ function calculateMenAverageAge(people, century) {
  * @return {number}
  */
 function calculateWomenAverageAge(people, withChildren) {
-  // write code here
+  let womens = people.filter(person => person.sex === 'f');
```

Comment:
The variable 'womens' should be named 'women' for grammatical correctness and use `const` instead of let